src/lumigo_tracer/event/event_trigger.py

```diff
@@ -256,6 +258,38 @@ def _parse_dynamomdb_event(event) -> Dict[str, Union[int, List[str]]]:
     }
 
 
-def _parse_sqs_event(event) -> Dict[str, Union[int, List[str]]]:
-    mids = [record["messageId"] for record in event["Records"] if record.get("messageId")]
-    return {MESSAGE_IDS_KEY: mids} if len(mids) > 1 else {MESSAGE_ID_KEY: mids[0]}
+def _parse_sqs_event(event) -> Dict[str, Union[int, str, List[str], List[Dict[str, str]]]]:
+    message_ids = []
+    chained_resources: List[Dict[str, str]] = []
+    for record in event.get("Records", []):
+        record_message_id = None
+        if record.get("messageId"):
+            record_message_id = record["messageId"]
+            message_ids.append(record_message_id)
+        body = record.get("body", "")
+        if (
+            record_message_id
+            and isinstance(body, str)
+            and "SimpleNotificationService" in body
+            and "TopicArn" in body
+        ):
```

Comment:
Please extract it to a function (or add comment)  because it's hard to understand what happens here, maybe something like `_is_sns(body)`.